### PR TITLE
Adds codefund banner in the issue explorer and issue details page

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description

Adds codefund banner with a link to codefund in the issue explorer page and bounty details.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

### CODE REVIEW

**Comment by SaptakS** [comment]
screen shot 2018-08-25 at 1 43 48 am

File: app/dashboard/templates/shared/codefund_banner.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<div class="w-100 mt-2" style="text-align: center;">
+  <a href="https://codefund.io?utm_medium=website&utm_source=gitcoin" target="_blank">
+    <img src="{% static "v2/images/codefund_banner.png" %}" style="max-width: 90%">
```
[the remark this answers is not in the document]
Comment (SaptakS) [reply app/dashboard/templates/shared/codefund_banner.html]: I would say 90% needed so as to always keep it within the width of the sidebar even if the width of the sidebar is variable in different page. For example in this case, the sidebar width for bounty details page is lesser than the issue explorer page. Hence.

**Comment by thelostone-mc** [comment]
@PixelantDesign could we get svg for it ?

**Comment by owocki** [comment]
lets make a note to track whether this converts or not please @coderberry -- if it converts, great, its worth the real estate.  but if not, we should remove it.. 

ill follow back in a month

**Pull request merged**